tuna/example/tables.py

```diff
@@ -34,17 +36,17 @@
 class ExampleDBTables(DBTablesInterface):
   """Represents db tables for example lib"""
 
-  def __init__(self, **kwargs):
+  def __init__(self, **kwargs: Dict[str, Any]) -> None:
     """Constructor"""
     super().__init__(**kwargs)
 
     #for pylint
-    self.job_table = None
-    self.session_table = SessionExample
+    self.job_table: Job = None
+    self.session_table: SessionExample = SessionExample
```

Comment:
Btw, since job_table and session_table are _types_ `Job` and `Whatever`, repsecitvely, and not _objects_ of these types, you need to do `self.job: typing.Type[Job] = Job`

Reply:
Personally I find it redundant. Having the constructor on the line is self explanatory to me. Having the name repeated doesn't help in my opinion.

Reply:
But there'd be no warning if I, by mistake, end up changing the variable's type:

```
self.session_table = SessionExample(...)
# ...
self.session_table = SubclassOfSessionMixin(...)
# ....
self.session_table = 2 # could be bad and go unnoticed
```

But if we have
``` 
self.session_table: SessionMixin = SessionExample(...)
# ...
self.session_table = SubclassOfSessionMixin(...)
# ....
self.session_table = 2 # mypy or smtg would complain that self.session_table can only be a SessionMixin
```

Reply:
I have defined Type